webapp/src/js/bootstrapper/translator.js

```diff
@@ -24,6 +28,10 @@ var TRANSLATIONS = {
     DOWNLOAD_APP: 'Descargando aplicación…',
     ERROR_MESSAGE: 'Error al cargar, por favor comprueba la conexión',
     TRY_AGAIN: 'Volver a intentar',
+    POLL_REPLICATION: 'Polling replication data…',
+    TOO_MANY_DOCS: ({ count }) => `Warning! You are about to download ${count} docs, which exceeds recommended limit. Do you wish to continue?`,
+    CONTINUE: 'Continue',
+    ABORT: 'Cancel',
```

Comment:
I'll request all of these translations after approval and final phrasing.

Reply:
Translations to be added to: https://docs.google.com/spreadsheets/d/1WxKJ2XMjTZopKHRUPGi9IIT4zKU2Q5-hyRjWx_c_nuI/edit?usp=sharing

Reply:
Translations updated.